src/action-ros-ci.ts

```diff
@@ -38,6 +38,13 @@ const curlFlagsArray = [
 	"--location",
 ];
 
+const validROS1Distros: string[] = ["kinetic", "lunar", "melodic", "noetic"];
+const validROS2Distros: string[] = ["dashing", "eloquent", "foxy"];
+const target_ros1_distro_input: string = "target-ros1-distro";
+const target_ros2_distro_input: string = "target-ros2-distro";
+const is_linux: boolean = process.platform == "linux";
+const is_windows: boolean = process.platform == "win32";
```

Comment:
I'm surprised this passed the linters.
I think the convention is camelCase for this repo (and most JS/TS projects I've seen).